schema/partner_shows.js

```diff
@@ -0,0 +1,40 @@
+import gravity from '../lib/loaders/gravity';
+import PartnerShow from './partner_show';
+import {
+  GraphQLString,
+  GraphQLList,
+  GraphQLInt,
+  GraphQLBoolean
+} from 'graphql'
+
+let PartnerShows = {
+  type: new GraphQLList(PartnerShow.type),
+  description: 'A list of PartnerShows',
+  args: {
+    size: {
+      type: GraphQLInt
+    },
+    sort: {
+      type: GraphQLString
```

Comment:
Oh cool, didn't realize we were using enums over just strings, will update! Thanks.